cmake/projects/taocpp-json/hunter.cmake

```diff
@@ -0,0 +1,31 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    taocpp-json
+    VERSION
+    1.0.0-beta.11-9163
```

Comment:
What 9163 means?

```
> git describe --tags e0895587
1.0.0-beta.11-645-ge0895587
```

So I expect something like `1.0.0-beta.11-e0895587`, see
- https://docs.hunter.sh/en/latest/creating-new/patch.html.

Reply:
Yep, absolutely correct. This was also my idea ... I copied/used the SHA1 :-(. Is fixed.